lib/sites/walmart.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const siteUtils = require('../site-utils');
+const logger = require('../logger')();
+
+class WalmartSite {
+  constructor(uri) {
+    if (!WalmartSite.isSite(uri)) {
+      throw new Error('invalid uri for Walmart: %s', uri);
+    }
+
+    this._uri = uri;
+  }
+
+  getURIForPageData() {
+    return this._uri;
+  }
+
+  isJSON() {
+    return false;
+  }
+
+  findPriceOnPage($) {
+    // find the price on the page
+    const priceString = $('*[itemprop="price"]').text();
```

Comment:
I might have spoken a bit too soon, I was looking at the TravisCI test output and saw this:

```
  price-finder:site-utils price string (lowercased):  $234.46  $234.46  +4ms
  price-finder:site-utils found dollar in price, converting to number... +0ms
  price-finder:site-utils price (post-process): 234.46234 +0ms
```

It looks like it's grabbing the price twice based on the wildcard in the selector.  We could do one of two things here to fix this:

1) Use `siteUtils.findContentOnPage` to process the multiple results, returning the first one:

```
    // the various ways we can find the price
    const selectors = [
      '*[itemprop="price"]',
    ];

    // find the price on the page
    const priceString = siteUtils.findContentOnPage($, selectors);
```

2) Narrow the selector to grab just one element.  Not quite sure how to handle this, since after looking at an example page, it looks like the content is just duplicated with some class changes.

What do you think?